### ISSUE TITLE
[Search] skillsetCount in ServiceCounters Shouldn't be Required

### ISSUE THREAD

**Issue by alzimmermsft** [Azure/azure-rest-api-specs#13963]
The current service preview Swagger for Search service added a new property to `ServiceCounters` as required.

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json#L9149

From an SDK perspective having this new parameter as required would cause SDKs generated on the latest Swagger to fail during deserialization if an older API version of Search was called. This new parameter should be optional, removed from being required.

**Comment by jennifermarsman** [comment]
To clarify, I assume that you are referring to "skillsetCount" as the new required property?  @jadrefke is investigating.

**Comment by alzimmermsft** [comment]
Yes, `skillsetCount` is the new property that is marked as required
